tests/passing-configs/index.js

```diff
@@ -14,8 +15,6 @@ export function filePathToConfigObj(filepath) {
   return {
     config,
     filepath,
-    name: path.parse(filepath).name,
```

Comment:
Our ESLint rules enforce [`comma-dangle`](http://eslint.org/docs/rules/comma-dangle). Could you please add the comma back and update this PR? Here are the commands:

```
git commit -a --amend --no-edit
git push -f
```

That will update your PR and keep only one commit in the history :+1